mods/ts/rules/nod-vehicles.yaml

```diff
@@ -133,10 +132,10 @@ ART2:
 		Range: 9c0
 	RenderSprites:
 	RenderVoxels:
+		NormalsPalette: ra2-normals
```

Comment:
```
# Voxel   Limb          NormalsType
art2      DUMMY01ING019 TiberianSun
art2tur   A_TURRET B    TiberianSun
art2barl  DUMMY01@      TiberianSun
```

This doesn't use RA2-style normals (afaik that older engine didn't understand them).